### DISCUSSION TITLE
use existing or reuse connection

### DISCUSSION THREAD

**Discussion by tmkontra** [tortoise/tortoise-orm#1595] [General]
from the docs, I am not able to determine if i would be able to either 
1) provide an existing psql connector/engine to Tortoise ORM
OR
2) reuse the psql connection pool from Tortoise ORM

for example, if I would like to use the same `psycopg` engine or pool for [Procrastinate](https://procrastinate.readthedocs.io/en/stable/howto/basics/connector.html#what-kind-of-connector-should-i-use)

is this possible?

**Comment by abondar** [comment]
Hi!

There is no way to make tortoise use other connection, created externally, but you can access tortoise connections through interface 
`from tortoise.connection import connections`

I think you just need `connections.get(<name of connection>)`, where name of connection is taken from config
